### DISCUSSION TITLE
test report for mobile testing template

### DISCUSSION THREAD

**Discussion by justmishka** [OWASP/mastg#2047] [Ideas]
Hi, 

as an output of my master thesis work I am going to generate the "report from testing template". I was thinking that this might be usefull also for you?

**Comment by cpholguera** [comment]
That sounds great @justmishka! Let us know more about that. If you'd like to join our Slack we can chat over there (instructions [here](https://github.com/OWASP/owasp-masvs/issues/622#issue-1116081917)). Else feel free to post here anything that might help or we search for an alternative channel. Děkuji!

**Reply by justmishka** [reply]
I joined the slack - "project-mobile_omtg" and reply in thread to the original issue. is that fine for you? I put there already some questions.

Thank you
